tools/github_actions_dependencies.sh

```diff
@@ -11,18 +11,18 @@ elif [ ! -z "$CONDA_DEPENDENCIES" ]; then
 else
 	echo "Install pip-pre dependencies"
 	test "${MNE_CI_KIND}" == "pip-pre"
-	python -m pip install $STD_ARGS pip setuptools wheel
+	python -m pip install $STD_ARGS pip setuptools wheel packaging
 	echo "Numpy"
 	pip uninstall -yq numpy
 	echo "Date utils"
 	# https://pip.pypa.io/en/latest/user_guide/#possible-ways-to-reduce-backtracking-occurring
 	pip install $STD_ARGS --pre --only-binary ":all:" python-dateutil pytz joblib threadpoolctl six
 	echo "PyQt6"
-	# Broken as of 2022/09/20
-	# pip install $STD_ARGS --pre --only-binary ":all:" --no-deps --extra-index-url https://www.riverbankcomputing.com/pypi/simple PyQt6
+	pip install $STD_ARGS --pre --only-binary ":all:" --default-timeout=60 --extra-index-url https://www.riverbankcomputing.com/pypi/simple PyQt6
 	pip install $STD_ARGS --pre --only-binary ":all:" PyQt6
```

Comment:
shouldn't the second line have been deleted?  Looks like installing PyQt6 twice here